docs/auth/README.md

```diff
@@ -7,6 +7,10 @@ By default, all data is inaccessible and you need to specify what resources can
 
 <iframe width="700" height="394" src="https://www.youtube.com/embed/o2ZN3nvdhi8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+::: warning
+Please, keep in mind that the access token is valid for 2 hours. 
```

Comment:
The word 'Please' doesn't need a comma after it. "Please keep in mind ..." is perfectly acceptable.

Reply:
will be fixed in next release

Reply:
Hi @MishaJexia, I've fixed this is #13 :+1: